tests/ui/target-feature/forbidden-hardfloat-target-feature-flag.rs

```diff
@@ -0,0 +1,12 @@
+//! Ensure ABI-incompaible features cannot be enabled via `-Ctarget-feature`.
+//@ compile-flags: --target=x86_64-unknown-linux-gnu --crate-type=lib
+//@ needs-llvm-components: x86
+//@ compile-flags: -Ctarget-feature=+soft-float
+// For now this is just a warning.
+//@ build-pass
+//@error-pattern: must be disabled
```

Comment:
Can we make this slightly more specific? "must be disabled" feels too likely to come up in other contexts.
```suggestion
//@error-pattern: must be disabled to ensure that the ABI
```